### PR TITLE
Addition of explanation of Disconnection Reasons

### PR DESCRIPTION
**Pull request by moskoweb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -178,6 +178,22 @@ type ConnectionState = {
...
+``` ts
+export enum DisconnectReason {
+	loggedOut           = 401, // Required Logout from Socket or User in the application.
+	connectionLost      = 403, // Connection was lost with WhatsApp or number was banned.
+	timedOut            = 408, // Waiting time for the very long connection.
+	multideviceMismatch = 411, // Incompatibility with multidevice.
+	connectionClosed    = 428, // Closed connection or without activities.
+	connectionReplaced  = 440, // Replaced connection, attempt of more than one equal connection.
+	badSession          = 500, // Bad connection or lost to the server.
+	restartRequired     = 515, // Required requirement for multidevice.
```
Comment (adiwajshing) [inline README.md:193]: required requirement?

Comment (moskoweb) [reply README.md:193]: As in some issues I saw several times repeating the question when each one is actually issued, I just wanted details to be clear, even though it is kind of logical what it is.

Comment (adiwajshing) [reply README.md:193]: I get that, but "required requirement" is a bit of an unnecessary emphasis on "requirement" no?

Comment (moskoweb) [reply README.md]: True, now re-reading everything, it got pretty ambiguous

File: README.md [hunk 2]
```diff
@@ -178,6 +178,24 @@ type ConnectionState = {
...
+	loggedOut            = 401, // Required Logout from Socket or User in the application.
+	connectionTerminated = 403, // Connection was terminated after number was banned.
+	connectionLost       = 408, // Connection was lost from Socket.
+	timedOut             = 408, // Waiting time for the very long connection.
+	multideviceMismatch  = 411, // Incompatibility with multidevice.
+	connectionClosed     = 428, // Closed connection or without activities.
+	connectionReplaced   = 440, // Replaced connection, attempt of more than one equal connection.
+	badSession           = 500, // Bad connection or lost to the server.
+	banned               = 503, // Number was banned from WhatsApp.
+	restartRequired      = 515, // Restart required for multiple devices.
```
Comment (adiwajshing) [inline README.md:195]: I think you mean "for multi-device WA"

**Pull request closed without merging**